deployment/deploy-local.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+#####################################################################
+# TRAP SHELL SIGNALS
+
+# clean-up on exit
+trap trap_signals 0 INT QUIT KILL TERM
+trap_signals() {
+    rm -rf /tmp/$$*
+}
+
+#####################################################################
+# PREFLIGHT CHECK
+
+if [ -z "$AWS_CLI_BIN" ]; then
+    AWS_CLI_BIN="awslocal"
+fi
+
+#####################################################################
+# VARIABLES
+
+outputs_file="/tmp/$$.cdk-infrastructure-outputs.json"
+
+#####################################################################
+# BUILD
+
+# build all lambdas first
+# do an incremental build during deployment.  this speeds up deployment time
+./build.sh incremental
+
+#####################################################################
+# DEPLOYMENT
+
+# log some info
+echo
+echo "Initiating CDK deployment.."
+
+# Bootstrap CDK
+cdklocal bootstrap aws://000000000000/us-east-1 || exit 4
+
+# Deployment for CDK stacks
+cdklocal deploy \
+    --require-approval never \
+    --outputs-file "${outputs_file}" \
+    --all || exit 5
+
+# Prepare web-ui deployment
+export COGNITO_USER_POOL_ID="$(awk '/UserPoolId/{ gsub(/"|,/, ""); print $2 }' "$outputs_file")"
```

Comment:
nit: We often use [`jq`](https://stedolan.github.io/jq/) to extract elements from a JSON file. (It's safe to assume that a lot of cloud devs have it installed, and we can also list it in the prerequisites in the README.) 

No need to change now - but maybe can make our lives a bit easier for the next samples.. 👍